keyboards/zykrah/fuyu/readme.md

```diff
@@ -0,0 +1,27 @@
+# FUYU
+
+![FUYU](https://i.imgur.com/uWB5EqJ.png)
```

Comment:
```suggestion
![FUYU](https://i.imgur.com/uWB5EqJh.png)
```

Reply:
Could I ask the reasoning behind this suggestion? For me, that link comes up empty on Imgur.